zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -407,3 +407,79 @@ class TuyaSoilSensor(CustomDevice):
             }
         },
     }
+
+
+class SoilManufClusterVar02(TuyaMCUCluster):
+    """Tuya Manufacturer Cluster with Temperature and Humidity data points (variation 02)."""
+
+    dp_to_attribute: dict[int, DPToAttributeMapping] = {
+        5: DPToAttributeMapping(
+            TuyaTemperatureMeasurement.ep_attribute,
+            "measured_value",
+            converter=lambda x: x * 10,
+        ),
+        3: DPToAttributeMapping(
+            TuyaSoilMoisture.ep_attribute,
+            "measured_value",
+            converter=lambda x: x * 100,
+        ),
+        15: DPToAttributeMapping(
+            TuyaPowerConfigurationCluster2AAA.ep_attribute,
+            "battery_percentage_remaining",
+            converter=lambda x: x * 2,  # double reported percentage
+        ),
+    }
+
+    data_point_handlers = {
+        3: "_dp_2_attr_update",
+        5: "_dp_2_attr_update",
+        15: "_dp_2_attr_update",
+    }
+
+
+class TuyaSoilSensorVar02(CustomDevice):
+    """Tuya temp and humidity sensor (variation 05)."""
```

Comment:
```suggestion
    """Tuya soil sensor (variation 2)."""
```
The other docstring is also wrong, can't add a suggestion for that here though. Feel free to PR the change for `TuyaSoilSensor`. It still says "temp and humidity sensor".

Reply:
The numbering seems a bit wrong overall. Feel free to improve it in a future PR.